MAUI/Segmented-Control/selection.md

```diff
@@ -0,0 +1,423 @@
+---
+layout: post
+title: Selection in .NET MAUI Segmented control | Syncfusion
+description: Learn about the Selection in Syncfusion .NET MAUI Segmented (SfSegmented) control in mobile and desktop applications from a single shared codebase.
+platform: maui
+control: SfSegmentedControl
+documentation: ug
+---
+ 
+# Selection in .NET MAUI Segmented (SfSegmented) control
+
+This section explains about features that aid in the selection of items, customization of the selected item, and its related operations in the Segmented Control.
+
+## Selected index
+
+You can set the default value programmatically for the selection to be placed. The selection gets updated based on the index value given for the `SelectedIndex`.
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage   
+    xmlns:buttons="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
+    <buttons:SfSegmentedControl x:Name="segmentedControl"
+                                SelectedIndex="1">
+    </buttons:SfSegmentedControl>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.SelectedIndex = 1;
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Ripple effect animation
```

Comment:
remove this
## Ripple effect animation

The Segmented Control supports ripple effect animation for selecting the item

Reply:
removed